egs/swbd/s5c/local/chain/tuning/run_tdnn_7m23h.sh

```diff
@@ -0,0 +1,519 @@
+#!/bin/bash
+
+# 7m23h is as 7m23b2 but with a small bugfix, removing a stray 'bottleneck-dim=192'.
```

Comment:
For anyone who is following this, this is the current best system-- just committing it for people's convenience.  Before I merge I will clean all this up.